Firmware/cardreader.h

```diff
@@ -130,13 +130,12 @@ class CardReader
   char filenames[SD_PROCEDURE_DEPTH][MAXPATHNAMELENGTH];
   uint32_t filesize;
   //int16_t n;
-  unsigned long autostart_atmillis;
+  ShortTimer autostart_atmillis;
   uint32_t sdpos ;
 
   bool autostart_stilltocheck; //the sd start is delayed, because otherwise the serial cannot answer fast enought to make contact with the hostsoftware.
   
-  uint16_t nrFiles; //counter for the files in the current directory and recycled as position counter for getting the nrFiles'th name in the directory.
-  char* diveDirName;
+  int16_t nrFiles; //counter for the files in the current directory and recycled as position counter for getting the nrFiles'th name in the directory.
```

Comment:
nrFiles should be uint16_t. It must have been reverted to int16_t 🤔

Reply:
https://github.com/prusa3d/Prusa-Firmware/pull/3245

Reply:
I hope other already merged PRs weren't affected during the rebase. If we see anything suspicious in this PR, we must investigate